js/src/mobx/hardwareStore.js

```diff
@@ -51,6 +52,21 @@ export default class HardwareStore {
     }, HW_SCAN_INTERVAL);
   }
 
+  scanTrezor () {
+    return this._api.parity
+      .trezor('get_devices')
+      .then((message) => {
+        this.pinMatrixRequest = JSON.parse(message).map((path) => {
```

Comment:
I'm not really a mobx expert, but shouldn't modification of this be done in some kind of `@action`? @ngotchac ?

Reply:
by adding an `@observable` and then in the module using it a `@observer`, as soon as there is a value change, the value updates. An action is not necessary.